hawkular-bus-common/src/main/java/org/hawkular/bus/common/MessageProcessor.java

```diff
@@ -338,42 +339,6 @@ public MessageId sendWithBinaryData(ProducerConnectionContext context, BasicMess
     }
 
     /**
-     * Same as {@link #sendRPC(ProducerConnectionContext, BasicMessage, Class, Map)} with <code>null</code> headers.
-     */
-    public <R extends BasicMessage> ListenableFuture<BasicMessageWithExtraData<R>> sendRPC(
```

Comment:
I think no one is going to use that. rather than just have things uncommented out everywhere, just remove this stuff entirely - the tests, samples, API. I thought some folks might want the ability to wait for a message to be processed and results returned (a-la RPC) but no one seems to want that.